cmake/onnxruntime_unittests.cmake

```diff
@@ -1205,4 +1219,12 @@ if (NOT onnxruntime_MINIMAL_BUILD AND NOT onnxruntime_EXTENDED_MINIMAL_BUILD
   endif()
 endif()
 
+if (onnxruntime_USE_STVM)
+  # find_library(STVM_LIBS NAMES libtvm.so PATHS ${onnxruntime_STVM_HOME}/lib)
+  # link_directories(onnxruntime_test_all ${STVM_LIBS})
+  find_library(PYTHON_LIBS NAMES libpython PATHS /usr/local/lib)
```

Comment:
why is python needed for unit tests?

Reply:
It is specific of TVM. It uses native and python code in combination.